scripts/migrations/8-state-record-data.py

```diff
@@ -0,0 +1,39 @@
+"""
+This migration implements StateRecord spec change by updating StateRecord::Data:
+https://github.com/nearprotocol/NEPs/pull/39
+
+Changes:
+- Removes `storage_paid_at` from Account
+- Modifies `StateRecord::Data` records
+"""
+
+import sys
+import os
+import json
+from collections import OrderedDict
+import base64
+
+home = sys.argv[1]
+output_home = sys.argv[2]
+
+config = json.load(open(os.path.join(home, 'output.json')), object_pairs_hook=OrderedDict)
+
+assert config['protocol_version'] == 7
+
+config['protocol_version'] = 8
+
+for record in config['records']:
+    if "Account" in record:
+        record["Account"]["account"].pop("storage_paid_at", None)
```

Comment:
I thought this is removed when we migrate to state rent?

Reply:
@bowenwang1996 it was not removed since we left (and still keep) those fields in the *View structures for compatibility reasons #2271